code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+const ThoughtForm = ({onSubmitThought, newThought, onNewThoughtChange}) => {
+    return (
+        <div className="input-container">
+            <p className="inputcard-text">What's making you happy right now?</p>
+            <form onSubmit={onSubmitThought}>
+                <label htmlFor="newThought"></label>
+                <textarea className="text-input-field"
+                    id="newThought"
+                    type="text"
+                    value={newThought}
+                    onChange={onNewThoughtChange}
+                    placeholder="type here"
+                    form="form"
+                >
+                </textarea>
+                <button 
+                    className="submit-button" 
+                    type="submit"
+                    disabled={newThought.length < 5 || newThought.length > 140}
```

Comment:
That's neat 😊. If you also want to prevent the user from writing more than 140 characters in the first place  "max-length" attribute could work. 🙂 Now I got inspired to add more attributes to my app as well. 😃